Corona-Warn-App/src/main/res/layout/include_settings_switch_row.xml

```diff
@@ -74,11 +74,11 @@
 
             </androidx.constraintlayout.widget.ConstraintLayout>
 
-            <Switch
+            <androidx.appcompat.widget.SwitchCompat
```

Comment:
could you please use `SwitchMaterial` instead?

Reply:
Does it make sense if the style is set to Widget.AppCompat.CompoundButton.Switch? I replaced it because AS was complaining.

Reply:
`SwitchMaterial` is actually a `SwitchCompat`, it will still work with `CompatTheme`. When we move to MD completely ,we will have to change the theme only

Reply:
I replaced it as requested